### PR TITLE
Added Witness benchmarking

### PR DESCRIPTION
**Pull request by Stephen-BCW**

### CODE REVIEW

**Review by daniel-bcw — APPROVED** [review]
We run it and then we get the CSV 👯

File: eth/stagedsync/stage_witness.go [hunk 1]
```diff
@@ -41,6 +44,73 @@ type WitnessStore struct {
 	GetHashFn       func(n uint64) libcommon.Hash
 }
 
+func logStageStart(stagestate string, blockNr uint64, logger log.Logger) time.Time {
+	start := time.Now()
+	logger.Info("`%s` on block %d starting at %s")
```
Comment (yan-bcw) [inline eth/stagedsync/stage_witness.go]: How do we pass the stagestate, blockNr in?

File: eth/stagedsync/stage_witness.go [hunk 2]
```diff
@@ -41,6 +44,73 @@ type WitnessStore struct {
 	GetHashFn       func(n uint64) libcommon.Hash
 }
 
+func logStageStart(stagestate string, blockNr uint64, logger log.Logger) time.Time {
```
Comment (yan-bcw) [inline eth/stagedsync/stage_witness.go]: stagestate - better use stageState?

File: eth/stagedsync/stage_witness.go (hunk 1 above)
Comment (Stephen-BCW) [reply eth/stagedsync/stage_witness.go]: It isn't really relevant here, but I wanted to log it, I just forgot to add it to the arguments of Info, thanks for catching that!

**Review by yan-bcw — APPROVED** [review]
LGTM

**Review by daniel-bcw — APPROVED** [review]

**Pull request merged**